parsers/CA_AB.py

```diff
@@ -116,24 +116,27 @@ def fetch_production(
     return validation.validate(
         {
             "capacity": {
-                "gas": generation["GAS"]["MC"],
-                "hydro": generation["HYDRO"]["MC"],
-                "battery storage": generation["ENERGY STORAGE"]["MC"],
-                "solar": generation["SOLAR"]["MC"],
+                "gas": generation["COGENERATION"]["MC"]
+                + generation["COMBINED CYCLE"]["MC"]
+                + generation["GAS FIRED STEAM"]["MC"]
+                + generation["SIMPLE CYCLE"]["MC"],
                 "wind": generation["WIND"]["MC"],
+                "solar": generation["SOLAR"]["MC"],
+                "hydro": generation["HYDRO"]["MC"],
                 "biomass": generation["OTHER"]["MC"],
-                "unknown": generation["DUAL FUEL"]["MC"],
-                "coal": generation["COAL"]["MC"],
+                "battery storage": generation["ENERGY STORAGE"]["MC"],
             },
             "datetime": get_csd_report_timestamp(response.text),
             "production": {
-                "gas": generation["GAS"]["TNG"],
-                "hydro": generation["HYDRO"]["TNG"],
-                "solar": generation["SOLAR"]["TNG"],
+                "gas": generation["COGENERATION"]["TNG"]
+                + generation["COMBINED CYCLE"]["TNG"]
+                + generation["GAS FIRED STEAM"]["TNG"]
+                + generation["SIMPLE CYCLE"]["TNG"],
                 "wind": generation["WIND"]["TNG"],
+                "solar": generation["SOLAR"]["TNG"],
+                "hydro": generation["HYDRO"]["TNG"],
                 "biomass": generation["OTHER"]["TNG"],
-                "unknown": generation["DUAL FUEL"]["TNG"],
```

Comment:
Has dual fuel been removed from the source?

Reply:
In previous years, fuel oil had no real value and coal was disappearing :
![image](https://github.com/user-attachments/assets/d8259690-f42c-49cb-a158-16f6debc9f6d)

The HTML page show only : 

- GAS
- Hydro
- Storage
- Solar
- Wind
- Biomass and other

no more fuel oil or coal seems to be used in Alberta, which seems consistent with the annual curves and certainly explains why they changed the data.

http://ets.aeso.ca/ets_web/ip/Market/Reports/CSDReportServlet?beginDate=&endDate=&contentType=html